### PR TITLE
docs: fix broken links

### PR DESCRIPTION
**Pull request by ml4**
## Description
Fix broken links when README is rendered in the public Registry.

## Related issue
[Issue #8](https://github.com/hashicorp/terraform-aws-terraform-enterprise-hvd/issues/8)

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update

## How has this been tested?
- [Describe the tests you ran to verify your changes]
- [Provide any results or outcomes from the tests]
- [Include any additional notes related to the tests]

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [-] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [-] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by roooms — APPROVED** [review]
Checked URLs and additional formatting

**Review by roooms — DISMISSED** [review]

**Review by roooms — APPROVED** [review]

File: docs/deployment-customizations.md [hunk 1]
```diff
@@ -54,9 +54,9 @@ ec2_os_distro = "<rhel>"
 ec2_ami_id    = "<custom-rhel-ami-id>"
```
Comment (dcanadillas) [inline docs/deployment-customizations.md:54]: ## Custom AMI
By default, this module will use the standard AWS Marketplace image based on the value of the `ec2_os_distro` input (either `ubuntu`, `rhel`, or `al2023`). 

```hcl
ec2_os_distro = "<rhel>"
```
If you prefer to use your own custom AMI, then you may do so by setting `ec2_ami_id` accordingly.
If you have a custom AWS AMI you would like to use, you can specify it via the following module input variables:
```hcl
ec2_ami_id    = "<custom-ami-id>"
```

File: docs/tfe-cert-rotation.md [hunk 2]
```diff
@@ -15,42 +15,45 @@ Follow these steps to rotate the certificates for your TFE instance.
 
 1. Obtain your new TFE TLS certificate file and private key file, both in PEM format.
 
-2. Update the values of the existing secrets in AWS Secrets Manager (`tfe_tls_cert_secret_arn` and `tfe_tls_privkey_secret_arn`, respectively). If you need assistance base64-encoding the files into strings prior to updating the secrets, see the examples below:
+1. Update the values of the existing secrets in AWS Secrets Manager (`tfe_tls_cert_secret_arn` and `tfe_tls_privkey_secret_arn`, respectively). If you need assistance base64-encoding the files into strings prior to updating the secrets, see the examples below:
 
     On Linux (bash):
+
```
Comment (dcanadillas) [inline docs/tfe-cert-rotation.md:21]: Would it be helpful to put an example on an AWS CLI command update using that base64? (just as an idea):

```bash
aws secretsmanager put-secret-value \
  --secret-id sandbox-tfe-tls-cert-base64 \
  --secret-string "{\"tfe_cert.pem\": \"$(cat new_tfe_cert.pem | base64 -w 0)\"}"
```

**Review by jrx — APPROVED** [review]

File: docs/tfe-cert-rotation.md (hunk 2 above)
Comment (roooms) [reply docs/tfe-cert-rotation.md:21]: Please could you submit these suggestions as new PRs or issues so we can consider them?

**Pull request merged**